pkg/controller/servicebindingrequest/binder_test.go

```diff
@@ -162,6 +164,71 @@ func TestBinderNew(t *testing.T) {
 	})
 }
 
+func TestBinderConfigMap(t *testing.T) {
+	ns := "binder"
+	name := "service-binding-request"
+	matchLabels := map[string]string{
+		"connects-to": "database",
+		"environment": "binder",
+	}
+	f := mocks.NewFake(t, ns)
+	sbr := f.AddMockedServiceBindingRequest(name, nil, "ref", "", deploymentsGVR, matchLabels)
+	f.AddMockedUnstructuredDeployment("ref", matchLabels)
+
+	sbr.Spec.BindingReference = &v1alpha1.BindingReference{
+		ObjectType: metav1.GroupVersionKind{
+			Version: "v1",
+			Kind:    "ConfigMap",
+		},
+	}
+
+	binder := NewBinder(
+		context.TODO(),
+		f.FakeClient(),
+		f.FakeDynClient(),
+		sbr,
+		[]string{},
+	)
+
+	require.NotNil(t, binder)
+
+	t.Run("appendEnvFrom-removeEnvFrom", func(t *testing.T) {
+		secretName := "secret"
```

Comment:
since the function is for testing binder config map, here `secretName` should be `configMapName`

Reply:
Thank you! Done https://github.com/redhat-developer/service-binding-operator/pull/366/commits/c8566ef6c187e26e9e987ffe184925e1e10b4fe0#diff-d422caad0c9d74644056915cb438c96a